### PR TITLE
feat: allow certificate to be supplied to `serve`

### PR DESCRIPTION
**Pull request by eduardoboucas**
**Which problem is this pull request solving?**

When the `certificatePath` option is supplied to the `serve` method, Edge Bundler will call `deno run` with the `--cert` flag.

Part of https://github.com/netlify/pillar-runtime/issues/323.

### CODE REVIEW

**Review by Skn0tt — APPROVED** [review]
looks mostly good to me, left a tiny question

**Comment by eduardoboucas** [comment]
> looks mostly good to me, left a tiny question

I don't see any comments 🤔

**Comment by Skn0tt** [comment]
hm, me neither - GitHub swallowed it? :shrug: give me a sec

File: src/server/server.ts [hunk 1]
```diff
@@ -56,7 +56,9 @@ const prepareServer = ({
       // no-op
     }
 
-    await deno.runInBackground(['run', ...flags, stage2Path, port.toString()], true, processRef)
+    const bootstrapFlags = ['--port', port.toString()]
```
Comment (Skn0tt) [inline src/server/server.ts:59]: looks like we didn't supply the `--port` flag before - assuming this is unrelated to the certificate path, was broken before, and you drive-by-fixed this.

Comment (eduardoboucas) [reply src/server/server.ts:59]: We did supply it, but as a positional parameter and not a flag (i.e. `deno run script.js <PORT>` as opposed to `deno run script.js --port <PORT>`). Using flags will allow us to supply additional parameters in the future in a more sustainable way.

You're right that it's not necessarily related to the certificate path, and I should've clarified that. Thanks for calling that out!

Comment (eduardoboucas) [reply src/server/server.ts:59]: See https://github.com/netlify/edge-functions-bootstrap/pull/30/files#r860951538.
